tests/resource/test_resource_getters.py

```diff
@@ -95,3 +98,29 @@ def test_resource_admin_get_my_access_raises_an_exception_when_extract_fails(
         )
         with self.assertRaises(SecurityRequestError):
             self.resource_admin.get_my_access("TESTING", "ELIJTEST")
+
+    # ============================================================================
+    # Auditing Rules
+    # ============================================================================
+    def test_resource_admin_get_audit_rules(
```

Comment:
Does it make sense to have a tests for extracting all possible values?

Reply:
Or is this all possible values?

Reply:
I don't know if there is an `all` key that we could end up with here.

Reply:
I can add more tests that pull out other combinations of values. All can also appear. Due to the nature of the interactions of the rules I have not yet generated a ruleset with ALL SUCCESS and FAILURE all at the same time, but I will try to do so to make this more comprehensive.

Reply:
So, if there are a few other permutations of audit rules that have the potential to break things, I definitely think it is worth looking into adding a couple more tests for those.

Reply:
upon further investigation of audit rules i'm not sure it is possible to have a "different" permutation. I can discuss test results offline but it looks like auditing at each access level is not just success:update means audit all successful update accesses, it means audit all successes update and above. Because of this, I don't think its possible to have successes, failures, and all meaningfully defined to be 3 different rules, so RACF will only ever really have 2.

The logic is that for any ALL rule, it must either supersede at least one rule by being about a "lower" level of access, or be superseded by the combination of both rules which together encompass "all" auditing for 2 lower levels of access.

As such, no additional tests to add.

Reply:
Maybe also the case where there are no audit rules as well.

Reply:
very fair point. I have added tests for that now.